File: 백준/Silver/2941. 크로아티아 알파벳/크로아티아 알파벳.py
a = input()

b = len(a)

for i in range(b):
    if a[i] == '=' and (a[i - 1] == 'z' and a[i - 2] == 'd'):
        b -= 1
    if a[i] == '=' and (a[i - 1] == 'c' or a[i - 1] == 's' or a[i - 1] == 'z'):
        b -= 1
    if a[i] == '-' and (a[i - 1] == 'c' or a[i - 1] == 'd'):
        b -= 1
    if a[i] == 'j' and (a[i - 1] == 'l' or a[i - 1] == 'n'):
        b -= 1

print(b)

File: 백준/Silver/2941. 크로아티아 알파벳/크로아티아 알파벳.js
const rl = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout,
});

rl.question('', (line) => {
    line = line.replace(/c=/g, 'A');
    line = line.replace(/c-/g, 'A');
    line = line.replace(/dz=/g, 'A');
    line = line.replace(/d-/g, 'A');
    line = line.replace(/lj/g, 'A');
    line = line.replace(/nj/g, 'A');
    line = line.replace(/s=/g, 'A');
    line = line.replace(/z=/g, 'A');

    console.log(line.length);

    rl.close();
});
